packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -83,8 +83,8 @@ const getCommonPlugins = (options: WebpackConfigurationOptions) => {
         { from: '../LICENSE', to: '.' },
         { from: 'img/*', to: '.' },
         { from: '**/*.json', to: '.' },
-        // { from: '**/*.svg', to: '.' },
-        // { from: '**/*.png', to: '.' },
+        { from: '**/*.svg', to: '.' },
+        { from: '**/*.png', to: '.' },
```

Comment:
Many existing plugins use 'images/' or 'imgs/' folder

Reply:
I see opportunity here introducing toolkit to set up a convention about where statics should belong in the plugin, see https://github.com/grafana/grafana/blob/master/packages/grafana-toolkit/README.md#working-with-css--static-assets. For those willing to have a different statics dir we could introduce some way of overriding default.

Reply:
In slack discussion, we agreed to keep this copy but promote /static in the docs.  We can deprecate this later, but for initial migration we do not want to require too much